modules/configuring-a-pipeline-server.adoc

```diff
@@ -55,8 +55,31 @@ If you specify incorrect data connection settings, you cannot update these setti
 +
 [NOTE]
 ====
-The external database connection is secured by default. To specify an external database connection that is not secured with TLS, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource as shown here:
+The external database connection is secured by default. To specify an external database connection that is not secured with TLS, after creating the Pipeline Server, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource:
```

Comment:
There is a difference between a custom resource and a custom resource definition. Looking at the procedure, I think you mean the latter.  See: https://docs.openshift.com/container-platform/4.15/operators/understanding/crds/crd-managing-resources-from-crds.html

Reply:
Addressed in latest commit.

Reply:
@syaseen-rh had it right the first time. What you are editing here is an instance of the Custom Resource Definition, and the instances of CRDs are called custom resources.

If you really wanted to edit the CRD, first, you can't or aren't supposed to, because the CRD is managed by an operator, and second, you'd do it on the YAML tab which shows the CRD itself, not through the Instances tab, which shows the CRs.

Reply:
Just a note, I'd do steps 1 and 2 through the Search screen instead. This is available both in the Administrator (under Home > Search) as well as in Developer perspective (Search).

![image](https://github.com/opendatahub-io/opendatahub-documentation/assets/442720/d60491d7-6112-4d10-aa21-45c4ec9fe5a4)